index.rb

```diff
@@ -126,6 +126,24 @@ def show(**kwargs)
   end
 end
 
+venues << Venue.new(:name => 'DNA Lounge', :link => 'https://www.dnalounge.com/calendar/dnalounge.rss') do
+  # Regex to dig the calendar link out of the description
+  link_regex = Regexp.new(/https:\/\/www.dnalounge.com\/calendar\/\d{4}\/[\d\w\-]+\.html/i)
+
+  URI.open(link) do |xml|
+    Nokogiri::XML(xml).css('item').map do |item|
+      description = item.css('description').text
+
+      show(
+        time: Time.parse(item.css('dnalounge|date').text),
```

Comment:
Will this include the time? Might have to parse `<dnalounge:time>`.

Reply:
Ah good call. I can fix this